dev-requirements-2.9.txt

```diff
@@ -5,6 +5,7 @@ ckantoolkit>=0.0.4
 factory-boy
 Faker
 flake8==3.8.3
+jinja2==2.10.1
```

Comment:
behaving is on 2.0.0, should be https://pypi.org/project/behaving/3.2.0/ 
can also update splinter.

Reply:
Updating behaving with CKAN 2.9 introduces a bunch of test failures.